src/test/CMakeLists.txt

```diff
@@ -244,7 +244,7 @@ set(
     testing_assert.cpp
     testing_assert.hpp
     utils/data_dependency_test_utils.hpp
-)
+    lib/logical_query_plan/logical_plan_root_node_test.cpp)
```

Comment:
Still not at the position it should be (files are ordered lexicographically in the CMakeLists.txt)